frontend/src/components/ViewActivities/editActivityFormElements.js

```diff
@@ -62,18 +61,45 @@ export function locationElementFormGroup(controlId, formLabel, dropdown) {
  * @returns {HTML} A FormGroup for date, time, and timezone.
  */
 export function dateTimeTzFormGroup(controlId, formLabel, dateRef,
-  dateDefault, timeRef, timeDefault, tzpicker) {
+  dateDefault, timeRef, timeDefault, tzpicker, show=true) {
+  const tzpickerElement = show ? tzpicker : (<div></div>);
   return (
     <Form.Group as={Row} controlId={controlId}>
       <Col sm={TITLEWIDTH}><Form.Label>{formLabel}</Form.Label></Col>
-      <Col sm={DATEWIDTH}>
+      <Col md="auto">
         <Form.Control type='date' label='date' ref={dateRef} defaultValue={dateDefault}/>
       </Col>
-      <Col sm={TIMEWIDTH}>
+      <Col md="auto">
         <Form.Control type='time' label='time' ref={timeRef}
           defaultValue={timeDefault}/>
       </Col>
-      <Col sm={TZPICKERWIDTH}>{tzpicker}</Col>
+      <Col sm={TZPICKERWIDTH}>{tzpickerElement}</Col>
     </Form.Group>
   );
 }
+
+/**
+ * Create a From Group with a checkbox.
+ * 
+ * @param {string} controlId FormGroup's control ID. 
+ * @param {string} formLabel Label of the field for this FormGroup. 
+ * @param {ref} ref The input's reference.
```

Comment:
I am not sure if I mentioned this before. But for refs, I think the correct type is `React.RefObject`